client/packages/host/webpack.config.js

```diff
@@ -144,6 +144,9 @@ module.exports = env => {
     plugins: [
       new ReactRefreshWebpackPlugin(),
       new webpack.DefinePlugin({
+        FEATURE_INVENTORY_ADJUSTMENTS: JSON.stringify(
+          env.FEATURE_INVENTORY_ADJUSTMENTS
+        ),
```

Comment:
Interesting, this actually does the opposite of what you want -- presumably you `JSON.stringify`-ed it to turn the string `"true"` into boolean `true`?

However, this ends up with `FEATURE_INVENTORY_ADJUSTMENTS: "true"` in `Environment` -- you just do it directly (`FEATURE_INVENTORY_ADJUSTMENTS: env.FEATURE_INVENTORY_ADJUSTMENTS,`) it makes it `true`.

I guess Webpack already handles "true" strings?